lib/bap/bap_inject_externals.mli

```diff
@@ -0,0 +1,9 @@
+open Bap_types.Std
+open Bap_image_std
+open Bap_sema.Std
+open Bap_disasm_std
+
+(** [run prg insn spec] performs search for relocations in [spec] and
+    add inject synthetic subroutines for each external function. Also
+    fixups calls to externals in [prg] *)
+val run : program term -> (mem * insn) seq -> Ogre.doc -> program term
```

Comment:
it is totally not obvious and is not reflected in the documentation what is the second argument for.